todo-app/src/todo.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import Description from './description'
+import TodoBox from './todoBox'
+import './App.css'
+
+class todo extends Component {
+    render() {
+        return (
+            <div class = 'todoList'>
```

Comment:
In React we should use `className` instead of `class` due to [jsx](https://reactjs.org/docs/introducing-jsx.html#specifying-attributes-with-jsx).

Reply:
@spirosikmd Thanks for feedback . It works here and completely fine in the browser . I don't know where is the problem . so I will read the code again